src/engraving/types/symnames.cpp

```diff
@@ -3087,3004 +3085,3007 @@ constexpr const std::array<AsciiStringView, size_t(SymId::lastSym) + 1> SymNames
     "space"
 } };
 
-constexpr const std::array<const char*, size_t(SymId::lastSym) + 1> SymNames::s_symUserNames { {
-    QT_TRANSLATE_NOOP("engraving/sym", "No symbol"),
+const std::array<muse::TranslatableString, size_t(SymId::lastSym) + 1> SymNames::s_symUserNames { {
+    muse::TranslatableString("engraving/sym", "No symbol"),
 
     // DO NOT edit the SMuFL standard symbol user names (see below) manually!
     // They are generated from tools/fonttools/smufl2sym.{bat,sh} and then
     // copied into this file! Edit tools/fonttools/smufl2sym-in-trans.json
     // to make more strings translatable!
```

Comment:
I just re-noticed this comment. Would be good to update the mentioned scripts; in practice, that seems to mean updating `/tools/fonttools/smufl2sym-cmd-symUserNames.jq`.